yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -97,7 +97,7 @@ func readSchemaFile(path string) []string {
 var sqlInfoCommentRegex = regexp.MustCompile("-- Name:.*; Type:.*; Schema: .*")
 
 // NOTE: This is for case when --schema-only option is provided with pg_dump[Data shouldn't be there]
-func parseSchemaFile(exportDir string, exportObjects string) int {
+func parseSchemaFile(exportDir string, exportObjectTypesList []string) int {
```

Comment:
Why take `exportObjectTypesList` as argument when the code in this function can directly access `source.ExportObjectTypesList`?